server/app/views/applicant/ApplicantProgramSummaryTemplate.html

```diff
@@ -5,14 +5,15 @@
   <body>
     <div th:replace="~{applicant/NavigationFragment :: pageHeader}"></div>
     <main>
+      <div
+        th:replace="~{applicant/TitleAndDescriptionFragment :: titleAndDescriptionFragment(${programTitle},${programDescription})}"
+      ></div>
       <div class="grid-container">
         <div class="grid-row">
           <div class="tablet:grid-col-8">
             <div
               th:replace="~{components/ToastFragment :: summaryToasts}"
             ></div>
-            <h1 th:text="${programTitle}"></h1>
-            <p th:text="#{content.reviewPageIntro}"></p>
```

Comment:
Non-blocking- Are we getting rid of the reviewPageIntro? I think it is good to have this step too.

Reply:
That's right, it is not present in the mocks: https://github.com/civiform/civiform/issues/7935